eZ/Publish/Core/MVC/Symfony/Controller/Content/ViewController.php

```diff
@@ -102,41 +99,9 @@ protected function buildResponse($etag = null, DateTime $lastModified = null)
      *
      * @deprecated Since 6.0.0. Viewing locations is now done with ViewContent.
      */
-    public function viewLocation($locationId, $viewType, $layout = false, array $params = array())
+    public function viewLocation(View $view)
```

Comment:
All changes here are introducing BC breaks since we documented to call the `ViewController` service when using custom controllers. So, as discussed already before we need to keep the old signatures and deprecate the old methods, and introduce new method(s) with the appropriate type hint.